pkg/v10/chartconfig/mock_test.go

```diff
@@ -6,7 +6,6 @@ import (
 	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
 	"github.com/giantswarm/helmclient"
 	"k8s.io/client-go/kubernetes"
-	"k8s.io/helm/pkg/helm"
```

Comment:
See https://github.com/giantswarm/cluster-operator/pull/356 I plan to merge this first.